.github/workflows/create-release.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: Create tag & release
+run-name: Create tag & release
+description: Create a new tag and release when a PR is merged to main (release/*)
+# yamllint disable-line rule:truthy
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+permissions:
+  contents: write
+jobs:
+  create_release:
+    if: >
+      github.event.pull_request.merged == true &&
+      github.event.pull_request.base.ref == 'main' &&
+      startsWith(github.event.pull_request.head.ref, 'release/')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@v4
+        with:
+          ref: main
+          fetch-depth: 0
+      - name: Get the version
+        id: get_version
+        run: |
+          NEW_TAG=$(cat .version)
+          PREVIOUS_TAG=$(git describe --tags --abbrev=0)
+          echo "NEW_TAG=${NEW_TAG}" >> ${GITHUB_ENV}
+          echo "PREVIOUS_TAG=${PREVIOUS_TAG}" >> ${GITHUB_ENV}
```

Comment:
How about we switch to using [Github Steps Output](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/passing-information-between-jobs) here ?